proto2python.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Bash script to create the python packages from the grpc proto for Armonik.Api
+# We are using the same structure than the C# package
+
+export PATH=$HOME/.local/bin:$PATH
+
+export PROTO_PATH="Protos/V1"
+export ARMONIK_PYTHON_SRC="Api/python/src" 
+export ARMONIK_SERVER=$ARMONIK_PYTHON_SRC"/armonik/server"
+export ARMONIK_CLIENT=$ARMONIK_PYTHON_SRC"/armonik/client"
+export ARMONIK_COMMON=$ARMONIK_PYTHON_SRC"/armonik/common"
+
+mkdir -p $ARMONIK_SERVER $ARMONIK_CLIENT $ARMONIK_COMMON
+
+# for debian/ubuntu:
+# sudo apt install python3-venv
+
+armonik_server_files=("agent_service.proto" "worker_service.proto")
+armonik_client_files=("submitter_service.proto" "tasks_service.proto" "sessions_service.proto")
+armonik_common_files=("objects.proto" "task_status.proto" "session_status.proto" \
+                      "result_status.proto" "agent_common.proto" "sessions_common.proto"  \
+                      "submitter_common.proto"  "tasks_common.proto"  "worker_common.proto")
+
+python -m pip install --upgrade pip
+python -m venv $HOME/grpc2
+python -m pip install build
+python -m pip install grpcio grpcio-tools fix-protobuf-imports
+
+for proto in ${armonik_server_files[@]}; do
```

Comment:
You should be able to generate with all files at once for each category

Reply:
done